src/plugins/irc-events/ctcp.js

```diff
@@ -1,6 +1,18 @@
 "use strict";
 
+const Helper = require("../../helper");
 const Msg = require("../../models/msg");
+const pkg = require("../../../package.json");
+
+const ctcpResponses = {
+	CLIENTINFO: () => Object // TOOD: This is currently handled by irc-framework
```

Comment:
`TOOD`?

Reply:
https://github.com/kiwiirc/irc-framework/issues/115

Reply:
I think @dgw was pointing out it should be TODO

Reply:
Fixed.